README.md

```diff
@@ -8,362 +8,63 @@ Blobfuse2 is stable and ***supported by Microsoft*** when used within its [docum
-* `gen-config` -  Auto generate recommended blobfuse2 config file.
 
 ## Find help from your command prompt
 To see a list of commands, type `blobfuse2 -h` and then press the ENTER key.
 To learn about a specific command, just include the name of the command (For example: `blobfuse2 mount -h`).
 
-## Usage
-- Mount with blobfuse2
-    * blobfuse2 mount \<mount path\> --config-file=\<config file\>
-- Mount blobfuse2 using legacy blobfuse config and cli parameters
-    * blobfuse2 mountv1 \<blobfuse mount cli with options\>
-- Mount all containers in your storage account
-    * blobfuse2 mount all \<mount path\> --config-file=\<config file\>
-- List all mount instances of blobfuse2
-    * blobfuse2 mount list
-- Unmount blobfuse2
-    * sudo fusermount3 -u \<mount path\>
-- Unmount blobfuse2 in lazy mode
-    * sudo fusermount3 -u \<mount path\> --lazy
-- Unmount all blobfuse2 instances
-    * blobfuse2 unmount all 
-- Auto generate config file
-    * blobfuse2 gen-config --tmp-path=\<local cache path\> --o \<path to save generated config\>
-    * blobfuse2 gen-config --block-cache --o \<path to save generated config\>
-
-<!---TODO Add Usage for mount, unmount, etc--->
-## CLI parameters
-- Note: Blobfuse2 accepts all CLI parameters that Blobfuse does, but may ignore parameters that are no longer applicable. 
-- General options
-    * `--config-file=<PATH>`: The path to the config file.
-    * `--log-level=<LOG_*>`: The level of logs to capture.
-    * `--log-file-path=<PATH>`: The path for the log file.
-    * `--foreground=true`: Mounts the system in foreground mode.
-    * `--read-only=true`: Mount container in read-only mode.
-    * `--default-working-dir`: The default working directory to store log files and other blobfuse2 related information.
-    * `--disable-version-check=true`: Disable the blobfuse2 version check.
-    * `--secure-config=true` : Config file is encrypted suing 'blobfuse2 secure` command.
-    * `--passphrase=<STRING>` : Passphrase used to encrypt/decrypt config file.
-    * `--wait-for-mount=<TIMEOUT IN SECONDS>` : Let parent process wait for given timeout before exit to ensure child has started. 
-    * `--block-cache` : To enable block-cache instead of file-cache. This works only when mounted without any config file.
-    * `--lazy-write` : To enable async close file handle call and schedule the upload in background.
-    * `--filter=<STRING>`: Enable blob filters for read-only mount to restrict the view on what all blobs user can see or read.
-    * `--preload`: Enable preload for read-only mount to start downloading all blobs from container when mount succeeds.
-- Attribute cache options
-    * `--attr-cache-timeout=<TIMEOUT IN SECONDS>`: The timeout for the attribute cache entries.
-    * `--no-symlinks=false`: By default symlinks will be supported and the performance overhead, that earlier existed, has been resolved.
-- Storage options

```

Comment:
Can you talk to Daniel, he has sort of automated this with the command help so that any new option we add, get automatically added to the documentation part.